src/main/scala/org/virtuslab/akkasaferserializer/CborAkkaSerializer.scala

```diff
@@ -0,0 +1,68 @@
+package org.virtuslab.akkasaferserializer
+
+import akka.serialization.Serializer
+import io.bullet.borer.{Cbor, Codec, Decoder, Encoder}
+
+import java.util.concurrent.atomic._
+import scala.reflect.ClassTag
+
+trait CborAkkaSerializer[Ser] extends Serializer {
```

Comment:
Pls use `BorerAkkaSerializer` consistently (instead of `Cbor`, since CBOR is also provided by e.g. Jackson)